frontend/user/configProfile/config_profile_mobile.html

```diff
@@ -0,0 +1,23 @@
+<md-content class="user-profile__container custom-scrollbar">
+    <div class="user-profile__header">
+        <div class="user-profile__header--background"></div>
+        <img class="user-profile__avatar user-profile__avatar--position" ng-src="{{ configProfileCtrl.photo_url }}">
+    </div>
+    <div class="user-profile__info">
+        <p class="user-profile__info--item user-profile__info--name responsive__title">{{configProfileCtrl.user.name}}</p>
+        <p class="user-profile__info--item"><span>e-mail: </span>{{configProfileCtrl.user.email[0]}}</p>
+        <p class="user-profile__info--item"><span>cpf: </span>{{configProfileCtrl.user.cpf}}</p>
+        <hr class="user-profile__info--divider"></hr>
+    </div>
+    <div class="user-profile__content">
+        <div class="user-profile__data" 
+            ng-repeat="prof in configProfileCtrl.user.institution_profiles track by prof.institution_key">
+            <img class="user-profile__avatar" ng-src="{{ prof.institution.photo_url }}"/>
+            <div>
+                <p class="responsive__title">{{prof.institution.name}}</p>
+                <user-profile-info office="prof.office" phone="prof.phone" email="prof.email"></user-profile-info>
+            </div>
+            <md-divider></md-divider>
```

Comment:
where is the edit button?

Reply:
I will add it when a do the edit profile page